### ISSUE TITLE
Sometimes the uix flashes pink

### ISSUE THREAD

**Issue by alexderpyfox** [Frooxius/NeosPublic#1749]
## Describe the bug
Sometimes when for example opening the dash or clicking on things in it like inventory tab or world browser it becomes flashes pink briefly

## Relevant issues
Not that I know
## To Reproduce
It just happens randomly 

## Expected behavior
Flashes pink once and then becomes normal 

## Screenshots / Video
I don't have one since it's hard to take a picture off 

## Bug information:
 - How often does it happen: sometimes 
 - Does the bug persist after restarting Neos? Yes
 - Neos Version: current version
 - Neos Platform: windows
 - Regression: it happens only with uix as I could tell 

## Reporters:
@alexderpyfox
@H3BO3

**Comment by Toxic-Cookie** [comment]
I've noticed this too. Just haven't found a solid replication case other than use the dash and it'll turn pink at times.

**Comment by H3BO3** [comment]
Please list the correct version in your report. "Current version" means nothing once an update gets pushed.

I've noticed this mainly during large data changes, less commonly when loading a heavy interface, but quite common when I leave a heavy world. Most elements on the interface just turn pink for a split second once I drop out of the world, but they reload fine.

**Comment by mralext20** [comment]
i believe this is a regression since the asset variance update for materials. 

no proof, just started noticing it when that update happened.

**Comment by shiftyscales** [comment]
Can everyone correlate any specific potential connections as far as graphics drivers/hardware? I don't think I've seen any such issues at all, and particularly not recently. I'm on a GTX 1070 with driver version 456.71.

**Comment by H3BO3** [comment]
I have a GTX1080Ti, driver 457.30

**Comment by mralext20** [comment]
rx 580, 8 GB. driver version 20.12.1

**Comment by alexderpyfox** [comment]
I got a 5800xt with driver 20.11.3 and I found out the pink flashing is not only uix but it does it to all objects in the world but not the world itself

**Comment by Frooxius** [comment]
This is very likely the same issue as this https://github.com/Frooxius/NeosPublic/issues/1499

It's been there for a while, it's caused by a race condition (essentially UI mesh gets updated before/after materials change), shouldn't be related to drivers or GPU at all.

It's not specifically harmful, so I haven't prioritized this yet, but I plan to look into it.

**Comment by alexderpyfox** [comment]
It also does it for every object tho not only ui I found out

**Comment by shiftyscales** [comment]
> 
> 
> This is very likely the same issue as this #1499
> 
> It's been there for a while, it's caused by a race condition (essentially UI mesh gets updated before/after materials change), shouldn't be related to drivers or GPU at all.
> 
> It's not specifically harmful, so I haven't prioritized this yet, but I plan to look into it.

Yeah- as was noted, this appears to affect objects in world/non-UI elements too, e.g. I PeterTheThinker mentioned the other day his avatar has occasionally momentarily flashed pink.

**Comment by AshtonSparx** [comment]
I've noticed this as well, usually during low frame-rate it seems to present itself more often.
